app/config/AppConfig.scala

```diff
@@ -38,6 +38,9 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: CTCServicesCon
   val routerUrl                = Url.parse(servicesConfig.baseUrl("transit-movements-router"))
   val auditingUrl              = Url.parse(servicesConfig.baseUrl("transit-movements-auditing"))
   val pushNotificationsUrl     = Url.parse(servicesConfig.baseUrl("transit-movements-push-notifications"))
+  val upscanInitiateUrl        = Url.parse(servicesConfig.baseUrl("upscan-initiate"))
+  val upscanCallbackUrl        = servicesConfig.config("upscan-initiate").get[String]("callbackUrl")
+  val upscanMaximumFileSize    = servicesConfig.config("upscan-initiate").get[Long]("maximumFileSize") * 1000000  // 1MB = 1000000 bytes
```

Comment:
This should be a long overall, but the config option should be a double if we're going to specify it in megabytes, just in case we want fractional megabytes.

Alternatively, we could change the configuration to be in kilobytes, and then leaving this as long (and a whole number) is viable (and thus probably more accurate). Happy either way, really.